src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/performance.html.twig

```diff
@@ -0,0 +1,274 @@
+                        <i class="material-icons">zoom_out_map</i> {{ 'CCC (Combine, Compress and Cache)'|trans }}
+                    </h3>
+                    <div class="card-block">
+                        <div class="card-text">
+
+                            {{ ps.infotip('CCC allows you to reduce the loading time of your page. With these settings you will gain performance without even touching the code of your theme. Make sure, however, that your theme is compatible with PrestaShop 1.4+. Otherwise, CCC will cause problems.'|trans({}, 'Admin.Advparameters.Help')) }}
+
+                            <div class="form-group">
+                                <label class="form-control-label">{{ 'Smart cache for CSS'|trans }}</label>
+                                {{ form_errors(combineCompressCacheForm.smart_cache_css) }}
+                                {{ form_widget(combineCompressCacheForm.smart_cache_css) }}
+                            </div>
+                            <div class="form-group">
+                                <label class="form-control-label">{{ 'Smart cache for JavaScript'|trans }}</label>
+                                {{ form_errors(combineCompressCacheForm.smart_cache_js) }}
+                                {{ form_widget(combineCompressCacheForm.smart_cache_js) }}
+                            </div>
+                            <div class="form-group">
+                                {{ ps.label_with_help(('Apache optimization'|trans), ('This will add directives to your .htaccess file, which should improve caching and compression.'|trans({}, 'Admin.Advparameters.Help'))) }}
+                                {{ form_errors(combineCompressCacheForm.apache_optimization) }}
+                                {{ form_widget(combineCompressCacheForm.apache_optimization) }}
+                            </div>
+                        </div>
+                    </div>
+                    <div class="card-footer">
+                        <button class="btn btn-primary">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
+                    </div>
+                </div>
+            </div>
+            {% endblock %}
+        </div>
+
+        <div class="row">
+            {% block perfs_form_media_servers %}
+            <div class="col">
+                <div class="card">
+                    <h3 class="card-header">
+                        <i class="material-icons">link</i> {{ 'Media servers (use only with CCC)'|trans }}
+                    </h3>
+                    <div class="card-block">
+                        <div class="card-text">
+
+                            {{ ps.infotip('You must enter another domain, or subdomain, in order to use cookieless static content.'|trans({}, 'Admin.Advparameters.Feature')) }}
+
+                            <div class="form-group">
+                                <label class="form-control-label">{{ 'Media server #1'|trans }}</label>
```

Comment:
Same comment as previously - is it possible to use a variable here to save some translation effort?